deploy/eck-kibana/values.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# Default values for eck-kibana.
+# This is a YAML-formatted file.
+
+# Overridable names of the Kibana instance.
```

Comment:
instance is a tricky word, here you mean Kibana later on in the comment on `count` you mean replicas when you say instance. We need to settle on consistent terminology. I personally would not mind if we called it just a Kibana resource in the context of Kubernetes, because that's what it is and name the replicas just replicas. That makes it easy for users familiar with K8s to map the concepts.

Reply:
I have gone through all files in the Charts looking for references to instance, or cluster `([k,K]ibana|[e,E]lasticsearch)\s(instance|cluster)` and adjusted them all to be consistently using `resource` instead.